translations/credits.tex/cs.po

```diff
@@ -78,6 +81,8 @@ msgid ""
 "\\textbf{Scenario templating and \\LaTeX{} writing}: Tomáš Zeman, Andrzej Wiącek, Saveliy Ivanov\n"
 "\n"
 msgstr ""
+"\\textbf{Šablony scénářů and \\LaTeX{} kód}: Tomáš Zeman, Andrzej Wiącek, Saveliy Ivanov\n"
```

Comment:
and?

Reply:
missed it... this one time where you don't use AI and this happens

Reply:
Ahhahah hope you learned your lesson 😆